sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/signature/KeyVaultKeyLessEcSha256Signature.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.security.keyvault.jca.implementation.signature;
+
+/**
+ * key vault SHA256
+ */
+public final class KeyVaultKeyLessEcSha256Signature extends KeyVaultKeyLessECSignature {
+
+    /**
+     * The default algorithm for certificate sign which Key Type is EC and Elliptic curve name is P-256 in key Vault will be used
```

Comment:
The default algorithm for certificate sign when the certificate is stored in Key Vault and Key Type is EC and Elliptic curve name is P-256.